### ISSUE TITLE
Docker compose gets a panick

### ISSUE THREAD

**Issue by Piste** [IgnisDa/ryot#178]
```log
ryot  | 2023-07-10T17:34:06.039759Z  INFO ryot::background: Running jobs after user creation
ryot  | thread 'tokio-runtime-worker' panicked at 'called `Result::unwrap()` on an `Err` value: ConnectionAcquire', apps/backend/src/miscellaneous/resolver.rs:2685:54
ryot  | note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```
Not sure what I'm doing wrong. 
Here's my docker compose snippet:
```
  ryot:
    image: 'ghcr.io/ignisda/ryot:latest'
    environment:
      - PUID=$PUID
      - PGID=$PGID
      - TZ=$TZ
      - WEB_INSECURE_COOKIE=true
    volumes:
      - $DOCKERDIR/appdata/ryot:/data
    ports:
        - '8888:8000'
    pull_policy: always
    container_name: ryot
```
Files are created in the `/data` directory

**Comment by IgnisDa** [comment]
What version of Ryot are you using?

**Comment by Finrodchen** [comment]
In my case, using the latest 1.8.3 version, got the same error message.

**Comment by IgnisDa** [comment]
Can you attach the entire logs of the container as well as your configuration, eg docker compose, database used etc.

**Comment by Finrodchen** [comment]
Thanks for your quick reply and your awesome work, I'd really love the app design and in my case I'm using portainer for docker management. but it's just like original docker-compose.

The ryot container setting below:
```
  ryot:
    image: 'ghcr.io/ignisda/ryot:latest'
    volumes:
      - $Portainer-docker-volume-path/ryot-data:/data
    ports:
        - '8090:8000'
    restart_policy: always
    container_name: ryot
```

And I setup a brand-new ryot-container, after user creation step, then received this error message.
```
2023-07-12T07:10:34.061469Z  INFO ryot: Listening on [::]:8000
2023-07-12T07:13:57.034349Z  INFO ryot::background: Running jobs after user creation
thread 'tokio-runtime-worker' panicked at 'called `R
[ryot_logs.txt](https://github.com/IgnisDa/ryot/files/12025074/ryot_logs.txt)
esult::unwrap()` on an `Err` value: ConnectionAcquire', apps/backend/src/miscellaneous/resolver.rs:2727:54
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

Then I added some media to watchlist and setting watch progress. And using the "Regenerate Summaries" to reproduce the error message.

Attach files include whole logs.

**Comment by IgnisDa** [comment]
I see you're using sqlite. Can you try with postgres?

**Comment by IgnisDa** [comment]
This is the error: https://docs.rs/sea-orm/latest/sea_orm/error/enum.DbErr.html#variant.ConnectionAcquire. I think SQLite is exhausting its connection pool.

**Comment by IgnisDa** [comment]
Released `v1.9.0` with a fix. Feel free to reopen if the issue persists.
